### PR TITLE
Added sub-section about internal load balancers

### PR DESCRIPTION
**Pull request by olvesh**
Had to dig through issues and source code to find this feature.
Is this enough or should it be expanded somewhat?

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3136)

### CODE REVIEW

File: docs/user-guide/services/index.md [hunk 1]
```diff
@@ -424,6 +424,22 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+#### Internal load balancer
+In a mixed environment sometimes it is necesarry to route traffic from services inside the same VPC (or equivivalent on non AWS platforms).
```
Comment (chenopis) [inline docs/user-guide/services/index.md]: Small grammar and spelling nits: "In a mixed environment`,` sometimes it is `necessary` to..."

**Comment by olvesh** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[docs/user-guide/services/index.md, line 428 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136#-KgTfo7ThfkD_N6H3B-6-r1-428:-KgpFlLwuy-zo4Hx2Ci4:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes.github.io/blob/dc5f42ed7fed360a8a65364b88156284a6270b8b/docs/user-guide/services/index.md#L428)):*
<details><summary><i>Previously, chenopis (Andrew Chen) wrote…</i></summary><blockquote>

Small grammar and spelling nits: "In a mixed environment`,` sometimes it is `necessary` to..."
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136)*

File: docs/user-guide/services/index.md [hunk 2]
```diff
@@ -425,7 +425,7 @@ an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is
 cloud provider does not support the feature, the field will be ignored.
 
 #### Internal load balancer
-In a mixed environment sometimes it is necesarry to route traffic from services inside the same VPC (or equivivalent on non AWS platforms).
+In a mixed environment sometimes it is necessarry to route traffic from services inside the same VPC (or equivalent on non AWS platforms).
```
Comment (chenopis) [inline docs/user-guide/services/index.md]: typo: "necessarry" => "necessary"

**Comment by chenopis** [comment]
Almost. =) Still "necessarry" => "necessary"

**Comment by olvesh** [comment]
Late night blindness. Fixed now.

---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136#-:-KgplGtnGX_uNhp-h2as:bkxa7ol)*

**Comment by olvesh** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[docs/user-guide/services/index.md, line 428 at r2](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136#-KgTfo7ThfkD_N6H3B-6-r2-428:-KgplV64k0e9MUzcvXpI:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes.github.io/blob/a908ddf30bc3fb40daffcefbb43ed8a074590d9f/docs/user-guide/services/index.md#L428)):*
<details><summary><i>Previously, chenopis (Andrew Chen) wrote…</i></summary><blockquote>

typo: "necessarry" => "necessary"
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136)*

**Comment by olvesh** [comment]
Moved to new doc structure

---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/3136#-:-KgrtPm367dYyJjWrkcc:b-w96rbi)*

**Comment by chenopis** [comment]
@bprashanth Can you take a look when you get a chance? Thx

**Comment by chenopis** [comment]
@bprashanth Ping.

File: docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -427,6 +427,21 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+#### Internal load balancer
+In a mixed environment it is sometimes necessary to route traffic from services inside the same VPC (or equivalent on non AWS platforms).
```
Comment (bowei) [inline docs/concepts/services-networking/service.md]: remove text in parenthesis()

File: docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -427,6 +427,21 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
...
+This can be achieved by adding the following annotation to the service:
```
Comment (bowei) [inline docs/concepts/services-networking/service.md]: add ##### AWS section and put AWS content under the section. This way, we have structure for other cloud providers when support is added.

Comment (olvesh) [inline docs/concepts/services-networking/service.md]: This is all AWS content. Even the annotation is AWS specific. IMO it makes more sense to  change section title to `Internal load balancer on AWS` that way it matches the AWS specific `SSL support on AWS` below.

**Comment by chenopis** [comment]
@bowei Will these changes work for you?

**Pull request merged**